roles/create_mce_crs/tasks/main.yml

```diff
@@ -0,0 +1,248 @@
+- name: Create Namespace for imported cluster
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "{{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ day2_add_worker_namespace }}"
+
+- name: Create Pull Secret
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "{{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+    definition:
+      apiVersion: v1
+      data:
+        .dockerconfigjson: "{{ docker_config }}"
+      kind: Secret
+      metadata:
+        name: pull-secret
+        namespace: "{{ day2_add_worker_namespace }}"
+      type: kubernetes.io/dockerconfigjson
+
+- name: Create InfraEnv
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "{{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+    definition:
+      apiVersion: agent-install.openshift.io/v1beta1
+      kind: InfraEnv
+      metadata:
+        name: "{{ day2_add_worker_namespace }}-infraenv"
+        namespace: "{{ day2_add_worker_namespace }}"
+        labels:
+          agentclusterinstalls.extensions.hive.openshift.io/location: Dallas
+          networkType: static
+      spec:
+        pullSecretRef:
+          name: pull-secret
+        sshAuthorizedKey: '{{ sshkey }}'
+        nmStateConfigLabelSelector:
+          matchLabels:
+            infraenvs.agent-install.openshift.io: "{{ day2_add_worker_namespace }}"

```

Comment:
@sarpkoksal Do we want to add something to template out the `network_config` to create the nmstate if required, or leave that to a future enhancement. If we leave it for later we need to add a inventory check to throw an error if `network_config` is defined.